ci-operator/step-registry/cucushift/upgrade/toimage/cucushift-upgrade-toimage-commands.sh

```diff
@@ -614,6 +622,12 @@ do
     if ! check_signed; then
         echo "You're updating to an unsigned images, you must override the verification using --force flag"
         FORCE_UPDATE="true"
+        if [[ "${ENABLE_OTA_TEST}" =~ "OCP-30832" ]] || [[ "${ENABLE_OTA_TEST}" =~ "OCP-27986" ]]; then
+            echo "ENABLE_OTA_TEST is enabled on this job, these cases need to run against a signed target image!"
+            exit 1
```

Comment:
exit 1 intentionally? do u want  the upgrade job with the two cases enabled to be dedicated for the cases?

Reply:
Yes, I did that intentionally, ideally we can add more cases for the checking, when we are going to add more cases to ensure upgrade without --force option, we can extend the list later.

Reply:
ack. it seems the same logic in [step](https://github.com/openshift/release/blob/610b94a984ac635f8dac65ff36a9ffad14de020c/ci-operator/step-registry/cucushift/upgrade/mirror-images/by-oc-adm/cucushift-upgrade-mirror-images-by-oc-adm-commands.sh#L156)  which is located before this step. so the job will exit in advance, and this exit will not be executed, right?

Reply:
Yeah, they are the same code, but the logic is applied to different case:
`cucushift-upgrade-mirror-images-by-oc-adm` ensure the mirroring payload should be signed, so that the following step signature configmap can be created successfully, or else, should exit in advance.
While `cucushift-upgade-toimage` step is ensure `oc adm upgrade` is run without `--force` option when the payload is signed, or else, exit immediately.

In the future, if we want to cover ipi behind proxy upgrade between stable builds, we just need to extend this list in `cucushift-upgade-toimage`, then we can easily cover all kinds of upgrade cases without --force option.